### ISSUE TITLE
Crash when trying to import one of the examples passes

### ISSUE THREAD

**Issue by sniperle** [ligi/PassAndroid#78]
```
04-19 18:46:05.969 I/PackageManager(815): Setting last chosen activity org.ligi.passandroid/.ui.PassImportActivity for user 0:
04-19 18:46:05.971 I/ActivityManager(815): START u0 {act=android.intent.action.VIEW dat=file:///storage/emulated/0/Download/charge_coupon.espass typ=*/* flg=0x13080000 cmp=org.ligi.passandroid/.ui.PassImportActivity} from uid 10150 on display 0
04-19 18:46:06.018 I/ActivityManager(815): Start proc 4005:org.ligi.passandroid/u0a202 for activity org.ligi.passandroid/.ui.PassImportActivity
04-19 18:46:06.084 W/System  (4005): ClassLoader referenced unknown path: /data/app/org.ligi.passandroid-1/lib/arm
04-19 18:46:06.326 I/PassAndroid(4005): Pass with same ID exists
04-19 18:46:06.411 I/ActivityManager(815): Displayed org.ligi.passandroid/.ui.PassImportActivity: +404ms
04-19 18:46:06.450 E/BitmapFactory(4005): Unable to decode stream: java.io.FileNotFoundException: /data/user/0/org.ligi.passandroid/files/passes/8cade31f-2a86-4d26-b212-c42a8a918fb2/bitmap@2x.png: open failed: ENOENT (No such file or directory)
04-19 18:46:06.451 E/BitmapFactory(4005): Unable to decode stream: java.io.FileNotFoundException: /data/user/0/org.ligi.passandroid/files/passes/8cade31f-2a86-4d26-b212-c42a8a918fb2/bitmap@2x.png: open failed: ENOENT (No such file or directory)
04-19 18:46:06.451 E/BitmapFactory(4005): Unable to decode stream: java.io.FileNotFoundException: /data/user/0/org.ligi.passandroid/files/passes/8cade31f-2a86-4d26-b212-c42a8a918fb2/bitmap@2x.png: open failed: ENOENT (No such file or directory)
04-19 18:46:06.451 E/BitmapFactory(4005): Unable to decode stream: java.io.FileNotFoundException: /data/user/0/org.ligi.passandroid/files/passes/8cade31f-2a86-4d26-b212-c42a8a918fb2/bitmap@2x.png: open failed: ENOENT (No such file or directory)
04-19 18:46:06.452 E/BitmapFactory(4005): Unable to decode stream: java.io.FileNotFoundException: /data/user/0/org.ligi.passandroid/files/passes/8cade31f-2a86-4d26-b212-c42a8a918fb2/bitmap@2x.png: open failed: ENOENT (No such file or directory)
04-19 18:46:06.452 E/BitmapFactory(4005): Unable to decode stream: java.io.FileNotFoundException: /data/user/0/org.ligi.passandroid/files/passes/8cade31f-2a86-4d26-b212-c42a8a918fb2/bitmap@2x.png: open failed: ENOENT (No such file or directory)
04-19 18:46:06.452 E/BitmapFactory(4005): Unable to decode stream: java.io.FileNotFoundException: /data/user/0/org.ligi.passandroid/files/passes/8cade31f-2a86-4d26-b212-c42a8a918fb2/bitmap@2x.png: open failed: ENOENT (No such file or directory)
04-19 18:46:06.452 E/BitmapFactory(4005): Unable to decode stream: java.io.FileNotFoundException: /data/user/0/org.ligi.passandroid/files/passes/8cade31f-2a86-4d26-b212-c42a8a918fb2/bitmap@2x.png: open failed: ENOENT (No such file or directory)
04-19 18:46:06.452 E/BitmapFactory(4005): Unable to decode stream: java.io.FileNotFoundException: /data/user/0/org.ligi.passandroid/files/passes/8cade31f-2a86-4d26-b212-c42a8a918fb2/bitmap@2x.png: open failed: ENOENT (No such file or directory)
04-19 18:46:06.453 E/BitmapFactory(4005): Unable to decode stream: java.io.FileNotFoundException: /data/user/0/org.ligi.passandroid/files/passes/8cade31f-2a86-4d26-b212-c42a8a918fb2/bitmap@2x.png: open failed: ENOENT (No such file or directory)
04-19 18:46:06.453 W/PassAndroid(4005): could not load pass.json from passcode 
04-19 18:46:06.457 D/PassAndroid(4005): Writing unhandled exception to: /data/user/0/org.ligi.passandroid/files/3.0.9-1461084366457.tracedroid
04-19 18:46:06.459 D/PassAndroid(4005): java.lang.IllegalArgumentException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.b, parameter pass
04-19 18:46:06.459 D/PassAndroid(4005): 	at org.ligi.passandroid.model.PassClassifier.moveToTopic(SourceFile)
04-19 18:46:06.459 D/PassAndroid(4005): 	at org.ligi.passandroid.ui.PassImportActivity$ImportAndShowAsyncTask$1.a(SourceFile:94)
04-19 18:46:06.459 D/PassAndroid(4005): 	at org.ligi.passandroid.ui.UnzipPassDialog$1AlertDialogUpdater$1$1.run(SourceFile:61)
04-19 18:46:06.459 D/PassAndroid(4005): 	at android.os.Handler.handleCallback(Handler.java:739)
04-19 18:46:06.459 D/PassAndroid(4005): 	at android.os.Handler.dispatchMessage(Handler.java:95)
04-19 18:46:06.459 D/PassAndroid(4005): 	at android.os.Looper.loop(Looper.java:148)
04-19 18:46:06.459 D/PassAndroid(4005): 	at android.app.ActivityThread.main(ActivityThread.java:5461)
04-19 18:46:06.459 D/PassAndroid(4005): 	at java.lang.reflect.Method.invoke(Native Method)
04-19 18:46:06.459 D/PassAndroid(4005): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
04-19 18:46:06.459 D/PassAndroid(4005): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
04-19 18:46:06.460 E/AndroidRuntime(4005): Process: org.ligi.passandroid, PID: 4005
04-19 18:46:06.460 E/AndroidRuntime(4005): 	at org.ligi.passandroid.model.PassClassifier.moveToTopic(SourceFile)
04-19 18:46:06.460 E/AndroidRuntime(4005): 	at org.ligi.passandroid.ui.PassImportActivity$ImportAndShowAsyncTask$1.a(SourceFile:94)
04-19 18:46:06.460 E/AndroidRuntime(4005): 	at org.ligi.passandroid.ui.UnzipPassDialog$1AlertDialogUpdater$1$1.run(SourceFile:61)
04-19 18:46:06.461 W/ActivityManager(815):   Force finishing activity org.ligi.passandroid/.ui.PassImportActivity
04-19 18:46:06.992 W/ActivityManager(815): Activity pause timeout for ActivityRecord{ea54394 u0 org.ligi.passandroid/.ui.PassImportActivity t15291 f}
04-19 18:46:08.061 I/WindowState(815): WIN DEATH: Window{7a300ad u0 org.ligi.passandroid/org.ligi.passandroid.ui.PassImportActivity}
04-19 18:46:08.064 I/WindowState(815): WIN DEATH: Window{b996f73 u0 org.ligi.passandroid/org.ligi.passandroid.ui.PassImportActivity}
04-19 18:46:08.097 I/ActivityManager(815): Process org.ligi.passandroid (pid 4005) has died

```

**Comment by ligi** [comment]
Thanks so much for reporting - I have a fix & roll out ASAP

**Comment by ligi** [comment]
fixed in 3.1.0
